components/form/form.directive.ts

```diff
@@ -55,6 +60,12 @@ export class NzFormDirective implements OnInit, OnChanges, AfterContentInit, OnD
     }
   }
 
+  updateItemsDefaultAutoTipConf(): void {
+    if (this.nzFormControlComponent) {
+      this.nzFormControlComponent.forEach(item => item.setDefaultAutoTipConf(this.nzTipOptions, this.nzDisableAutoTips));
+    }
+  }
+
   constructor(public nzConfigService: NzConfigService, elementRef: ElementRef, private renderer: Renderer2) {
     this.renderer.addClass(elementRef.nativeElement, 'ant-form');
```

Comment:
miss 
   this.nzConfigService
      .getConfigChangeEventForComponent(NZ_CONFIG_COMPONENT_NAME)
      .pipe(takeUntil(this.destroy$))
      .subscribe(() => {
        this.cdr.markForCheck();
      });